config/config.go

```diff
@@ -1,13 +1,63 @@
 package config
 
-// DebugMode contains boolean of whether application is being run in development
-var DebugMode bool
+import (
+	"github.com/ONSdigital/go-ns/log"
+	"github.com/kelseyhightower/envconfig"
+)
 
-// SiteDomain is the current domain that the router is hosted on
-var SiteDomain = "ons.gov.uk"
+// Configuration structure which hold information for configuring the import API
+type Config struct {
+	BindAddr                 string    `envconfig:"BIND_ADDR"`
+	Debug                    bool      `envconfig:"DEBUG"`
+	SiteDomain               string    `envconfig:"SITE_DOMAIN"`
+	PatternLibraryAssetsPath string    `envconfig:"PATTERN_LIBRARY_ASSETS_PATH"`
+	SupportedLanguages       [2]string `envconfig:"GENERATE_DOWNLOADS_TOPIC"`
+	EnableCookiesControl     bool      `envconfig:"ENABLE_COOKIES_CONTROL"`
```

Comment:
Question: Is this the same value across each app?

Reply:
This is the lift and shift from the dp-frontend-dataset-controller cookie flag. (This was the one as a cookie team we had agreed on). However, I have seen one elsewhere with a slightly different name, not ideal but for this app I think it is best to stick to the agreed name.